scripts/version

```diff
@@ -2,4 +2,4 @@
 set -e
 
 CHARTS_BUILD_SCRIPTS_REPO=https://github.com/rancher/charts-build-scripts.git
-CHARTS_BUILD_SCRIPT_VERSION=v0.2.1
\ No newline at end of file
+CHARTS_BUILD_SCRIPT_VERSION=v0.0.0-dev
```

Comment:
To be bumped to v0.2.2 when https://github.com/rancher/charts-build-scripts/pull/57 is merged and a release is cut. This should fix the build workflow failure as well.